src/applets/night-light/com.solus-project.night-light.gresource.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<gresources>
+        <gresource prefix="/com/solus-project/night-light">
```

Comment:
For new applets please use `/org/budgie-desktop` - we can't move the old ones but no need to make it stick on the new ones.